util/reggen/fpv_csr.sv.tpl

```diff
@@ -86,6 +86,14 @@ module ${mod_base}_csr_assert_fpv import tlul_pkg::*;
   // accesses).
   bit [${hro_idx_width-1}:0] hro_idx;
   bit [${addr_msb}:0] normalized_addr;
+  % if lblock == "clkmgr":
+    % for hro_reg in hro_regs_list:
+      % if hro_reg.name.lower() == "jitter_enable":
```

Comment:
Just a note here that while having code specific to a block and register hard-coded into reggen isn't a scalable or future-proof solution, I think it's totally appropriate for implementing this ECO on the `earlgrey_1.0.0` branch.